gui/default/syncthing/core/syncthingController.js

```diff
@@ -97,14 +104,30 @@ angular.module('syncthing.core')
             files: 0
         };
 
+        $scope.isDefaultLoginOption = function (loginOption) {
+            if (window.localStorage) {
+                return (window.localStorage.getItem("syncthing-default-login-option") || 'password') === loginOption;
+            } else {
+                return loginOption === 'password';
+            }
```

Comment:
Simplify by setting fallback first (untested diff):

```suggestion
            var defaultSetting = 'password';
            if (window.localStorage) {
                defaultSetting = window.localStorage.getItem("syncthing-default-login-option") || defaultSetting;
            }
            return loginOption === defaultSetting;
```

Reply:
Suggestion applied in 90eed7fb00d7bd03d1ce0a3abf1f6f51a3e21985.